src/components/UserManagement/UserManagement.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { px } from '../../utils';
+import Input from '../Input/Input';
+import AccountCircleGrey from '../../assets/account_circle_grey.svg?react';
+import NavigationItemTrigger from '../Navigation/NavigationItemTrigger/NavigationItemTrigger';
+import NavigationList from '../Navigation/NavigationList/NavigationList';
+import NavigationItem from '../Navigation/NavigationItem/NavigationItem';
+
+export type UserManagementProps = React.HTMLAttributes<HTMLElement>;
+
+const UserManagement = ({ children }: React.PropsWithChildren<UserManagementProps>) => {
+  const [language, setLanguage] = React.useState('English');
+
+  return (
+    <div data-testid="user-management" className={`${px}-user-management`}>
+      <span className={`${px}-user-management__account-wrapper`}>
+        <AccountCircleGrey className={`${px}-user-management__account-icon`} />
+        <NavigationItem href="#login" label={`Login`} />
+      </span>
+      <NavigationItemTrigger className={`${px}-user-management__language`} label={language}>
+        <NavigationList className={`${px}-user-management__language__selections`}>
+          <li>
+            <Input
+              type="radio"
+              id="radio-en"
+              labelText="English"
+              value="en"
+              inline
+              name="languages"
+              defaultChecked
+              onClick={() => setLanguage('English')}
+            />
+          </li>
+          <li>
+            <Input
+              type="radio"
+              id="radio-zh"
+              labelText="中文"
+              value="zh"
+              inline
```

Comment:
Should we make supported languages a prop that we can map over if additional languages become supported in the future?

Reply:
Good call. I'll update this to be more flexible